### ISSUE TITLE
Quick Fix not working for local imports

### ISSUE THREAD

**Issue by eronning** [microsoft/pylance-release#3353]
## Environment data

-   Language Server version: Python
-   OS and version: macOS Monterey
-   Python version (& distribution if applicable, e.g. Anaconda): 3.10.4
-   Pylance version: v2022.9.20

## Comments

I know other people have run into issues with being unable to auto-import locally, which is unavailable ([as per this issue](https://github.com/microsoft/pylance-release/issues/3324)). However, to my understanding, you should be able to hover over local imports and quick-fix auto-add the import. I was recently able to do this until I updated my VSCode and extensions recently.

## Repro Steps

1. Install VSCode with the following extensions:

- Python
- Pylance
- Intellicode

2. Create a new python project folder with a venv using poetry.
3. Place the following in the settings.json:

```
{
  "python.envFile": "${workspaceFolder}/.env",
  "python.languageServer": "Pylance",
  "python.analysis.indexing": true,
  "python.formatting.provider": "black",
  "python.analysis.autoImportCompletions": true,
  "python.analysis.autoSearchPaths": true,
  "python.autoComplete.extraPaths": ["~/Development/<django repo name>/server"],
  "python.analysis.extraPaths": ["~/Development/<django repo name>/server"],
  "vsintellicode.features.python.deepLearning": "enabled",
}
```

## Expected behavior
Currently, I can quick-fix to auto import python modules that I have defined in my poetry config (and installed in the venv). See the image for an example. I should be able to do the same for functions/classes that I have defined locally (and are in files that are opened in VS Code).

<img width="770" alt="Screen Shot 2022-09-16 at 2 31 31 PM" src="https://user-images.githubusercontent.com/10899165/190709038-a5eecc89-bebf-4dc9-8d95-67d1286e58d4.png">

## Actual behavior
I am unable to quick-fix import local classes and functions. 

Say I define a dataclass of `SampleDataClass` in my `dataclasses.py` file like so:
<img width="370" alt="Screen Shot 2022-09-16 at 2 27 42 PM" src="https://user-images.githubusercontent.com/10899165/190711015-c01825f3-bd6a-4978-8746-31ba70983c18.png">

Then I attempt to reference that dataclass in a new file called `test_file.py` located in the same directory. With both files open I'm still unable to hover over the reference and auto-import the dataclass. See below:
<img width="726" alt="Screen Shot 2022-09-16 at 2 28 22 PM" src="https://user-images.githubusercontent.com/10899165/190711058-50b9ea88-7e31-483b-b24f-8faaf84aa6bd.png">

**Comment by heejaechang** [comment]
It should work. 

Can you provide us some logs as described in [the troubleshooting guide](https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md#filing-an-issue)?

I would like to see whether we are correctly picking up the file. (do we consider the file as user file?)

**Comment by wf-tomluci** [comment]
Hello, same issue here on Pylance v2022.9.10 and v2022.9.20 and most recently Pylance v2022.9.10 and v2022.9.30.

I've tested Pylance v2022.8.50 and for that version the import suggestions do work.

Here are my details, as per the [Filing an issue](https://github.com/microsoft/pylance-release/blob/main/TROUBLESHOOTING.md#filing-an-issue) guide:
- Language Server version: Pylance v2022.9.1, v2022.9.20 and v2022.9.30
  - Output: `Pylance language server 2022.9.30 (pyright df789b5e) starting`
- OS and version: macOS Monterey 12.5.1
- Python version: 3.10.4

> A code example (or any other additional information) we can use to reproduce the issue.

The example provided by `eronning` is perfect.

Other extensions installed:

- Python `v2022.14.0`
- Django `v1.10.0`
- IntelliCode `v1.2.27`

**Comment by eronning** [comment]
I've attached some logs for that version. However, it looks like if I either roll back to `v2022.8.50` or roll forward to the latest version of `v2022.9.40` the issue has been resolved. 

[logs.txt](https://github.com/microsoft/pylance-release/files/9675422/logs.txt)

**Comment by heejaechang** [comment]
Thank you @eronning for letting us know that it works with the latest pylance and providing us the log.

according to your log file

```
[Info  - 10:07:45 AM] (7144) Searching for source files
[Info  - 10:07:45 AM] (7144) No source files found.
```

you seem hitting an old bug that has been fixed in 2022.9.40 (https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2022940-26-september-2022-release)

this issue (Bug Fix: Default includes and excludes weren't being used if a pyproject.toml file was present (https://github.com/microsoft/pylance-release/issues/3348)) particularly. 

since now it is resolved, I will close the issue. thank you.

**Comment by Vorobeyko** [comment]
@heejaechang Hi!
I have a same problem(( 
<img width="388" alt="image" src="https://github.com/microsoft/pylance-release/assets/11293201/d7461607-774f-432d-892d-ca367a89bb67">

I have a class KafkaConsumers but i cannot import it througth quickfix.

What i doing incorrect?
